pywbem_os_setup.sh

```diff
@@ -0,0 +1,327 @@
+#!/bin/bash
```

Comment:
Are you sure you want to check in this file? I would suggest to exclude it in .gitignore, instead.

Reply:
Yes. It has one fix needed for that shell file.